src/uuid/builder/uuidutils.rs

```diff
@@ -31,8 +33,8 @@ impl UuidUtils {
     /// # Returns
     ///
     /// * `Vec<u8>` - The byte array representation of the UUID
-    pub fn to_bytes(uuid: &Uuid) -> Vec<u8> {
-        uuid.as_bytes().to_vec()
+    pub fn to_bytes(uuid: &uproto_Uuid) -> Vec<u8> {
+        Uuid::from(uuid.clone()).as_bytes().to_vec()
```

Comment:
I'm not 100% sure, but I seems there is some extra copy here that could be avoided.

Reply:
done